### ISSUE TITLE
new_proj() - line by line text entry prone to mistakes - how do we undo?

### ISSUE THREAD

**Issue by jzadra** [IntegralEnvision/integral#11]
Since the new_proj() is prone to mistakes that aren't easily reversible, we might consider:

1) A confirmation at the end of the process that allows modification
2) A function to undo creating the project (probably bad)
3) A function to update a project

**Comment by ebenp** [comment]
Maybe an ask / confirm repeat workflow when used interactively?

**Comment by jzadra** [comment]
Yeah.  Perhaps the selections/values could be listed with the option to select the number to edit, or just enter to confirm.  (As opposed to having to start from scratch and re-enter everything when only one thing was incorrect).

**Comment by ebenp** [comment]
Currently abandoned in current version. Maybe just have them rerun?
